src/com/ms/silverking/cloud/dht/daemon/storage/NamespaceStore.java

```diff
@@ -723,8 +723,8 @@ private long newestVersion(DHTKey key) {
     
         newestSegment = newestSegment(key);
         if (debugVersion) {
-            System.out.printf("newestVersion(%s)\n", key);
-            System.out.println("newestSegment: " + newestSegment);
+            Log.fineAsyncf("newestVersion(%s)", key);
```

Comment:
debugVersion is off by default. Make Log.warningAsyncf so that the debugVersion flag has an effect